examples/hovorod-cpu/Dockerfile.cpu

```diff
@@ -0,0 +1,78 @@
+FROM ubuntu:18.04
+
+ENV TENSORFLOW_VERSION=1.14.0
+ENV PYTORCH_VERSION=1.4.0
+ENV TORCHVISION_VERSION=0.5.0
+ENV MXNET_VERSION=1.6.0
+
+# Python 2.7 or 3.6 is supported by Ubuntu Bionic out of the box
+ARG python=2.7
```

Comment:
Can we use Python 3.6 then? Have you tested with Python 3.6?

Reply:
Just run some tests with Python3.6, it works fine ;)

Reply:
I have set Python3.6 as the default  in Dockerfile.